### DISCUSSION TITLE
Problems with short utterances of numbers.

### DISCUSSION THREAD

**Discussion by oscarhinde** [deepgram/community#495] [General help]
We're having trouble with the following specific scenario: Our voice assistant sometimes asks questions where the person is expected to respond with a number within a scale (e.g. 1 to 3 or 1 to 5). Our clients usually respond with monosyllables ("four", "two") and Deepgram is often not catching the numbers properly and instead transcribing "hi", "hey", "good" and other common words. We've tried keywords but I suppose numerals fall into the "common word" category and are therefore being ignored as valid keywords. Our product processes audio streamed live from phonecalls over voip. We've tried both nova-2 and nova-2-phonecall to no avail. Has anyone had issues like this and are there any specific guidelines for this usecase?

**Example**
User says "three" but the transcription result is "hi".

Request ID: 5a653daf-384e-44d8-8d2d-6a246a022a29
POST /v1/listen?tier=nova&model=2-phonecall&language=en-US&encoding=linear16&sample_rate=16000&smart_format=false&punctuate=true&dates=true&date_format=%25d%2F%25m%2F%25Y&measurements=false&filler_words=false&profanity_filter=false&interim_results=true&endpointing=1000&tag=********

Tag is obfuscated for security reasons

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @oscarhinde, thanks for writing in, and I'm sorry you're having this problem with Deepgram.

I looked at the request ID you provided, and that particular audio is challenging. I hear "hi", but if I was told it was definitely a number, perhaps I'd guess "five".

That being said, I've heard other customers report similar issues to the one you're describing. If you're able to share additional request IDs, I can look further with a broader scope of requests.

I'd also suggest trying our `nova-phonecall` model, as you may observe different behavior than with the `nova-2` and `nova-2-phonecall` models. One more possibility is to try our older `base` or `enhanced` models, which generally have lower overall accuracy in our reporting metrics, but can be preferable in certain conditions, potentially such as the one you're describing with a narrow vocabulary and lower audio quality.
